themes/mongodb/src/js/componentEcosystemLinkTrack.js

```diff
@@ -0,0 +1,20 @@
+import {reportAnalytics, toArray} from './util';
+
+function reportClick(anchorElement) {
+    reportAnalytics('Link Clicked', {
+        'text': anchorElement.text,
+        'href': anchorElement.href,
+    })
+}
+
+export function setup() {
+    // only on ecosystem homepage, track link clicks
+    if (document.body.dataset && document.body.dataset.project === 'ecosystem' && window.location.href === 'https://docs.mongodb.com/ecosystem/') {
```

Comment:
Y'know, thinking about it, isn't this redundant? If you're checking href, then that includes the project.